zkit/src/main.rs

```diff
@@ -205,6 +205,8 @@ struct StarkProveOpt {
     stark_struct: String,
     #[clap(short = "p", long = "piljson", default_value = "pil.json")]
     piljson: String,
+    #[clap(short = "n", long = "norm_stage", default_value = "false")]
```

Comment:
use `action=ArgAction::SetFalse` to disable it by default.  Meanwhile, I suggest declaring `norm_state` type as bool, instead of string.

Reply:
Thus we also don't need `norm_state.parse().unwrap()`

Reply:
I will fix it with a new pr together with upgrading clap from 3.0 to 4.3.4